lib/features/home/data/datasource_impl/session_datasource_impl.dart

```diff
@@ -13,7 +14,9 @@ class SessionDataSourceImpl extends SessionDataSource {
   @override
   Future<Session> getSession() {
     return Future.sync(() async {
-      return await _sessionAPI.getSession();
+      return await _sessionAPI.getSession(
+        converters: SessionExtensions.customMapCapabilitiesConverter,
```

Comment:
we should add `defaultConverter` too?

Reply:
By default there is already a `defaultConverter`, we just add it to the existing map.